src/app/modules/entry-live/views/live-discovery-chart/discovery-chart/discovery-chart.component.ts

```diff
@@ -238,6 +235,11 @@ export class DiscoveryChartComponent {
       ],
     };
   }
+
+  private shouldNormalize(metric1, metric2): boolean {
```

Comment:
please add typings for the function's params

Reply:
done